Apps/WebClient/src/ClientApp/src/components/modal/verifySMS.vue

```diff
@@ -10,6 +10,7 @@ import { ILogger, IUserProfileService } from "@/services/interfaces";
 import UserSMSInvite from "@/models/userSMSInvite";
 import type { WebClientConfiguration } from "@/models/configData";
 import { DateWrapper } from "@/models/dateWrapper";
+import moment from "moment";
```

Comment:
We are not using moment anymore

Reply:
How come? It's super useful for a lot of applications is it not?

Reply:
nvm